.deployment.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "name": "duxcore-backend",
```

Comment:
i think naming it just duxcore is more suitable since we might want to host all containers under the same env

Reply:
oh yeah, I forgot my bad...